lib/rubygems/commands/cleanup_command.rb

```diff
@@ -10,10 +10,16 @@ def initialize
           :force => false, :install_dir => Gem.dir,
           :check_dev => true
 
+    add_option('-n', '-d', '--dry-run',
+               'Do not uninstall gems') do |value, options|
+      options[:dryrun] = true
+    end
+
```

Comment:
What happens with the `-n` and `-d` shortcuts? Do they warn? I think they shouldn't.

Also (althought unrelated to this PR but since we're here), why two shortcuts? Seems like an incorrect usage of `add_option` :thinking:.

Reply:
`-d` and `-n` will not warn.

> Also (althought unrelated to this PR but since we're here), why two shortcuts? Seems like an incorrect usage of add_option

🤔  I don't know what was the reasoning for those two shortcuts, but I agree with you I think it should be just one